src/main/kotlin/oop/Proxy/ProtectionProxy.kt

```diff
@@ -1,4 +1,4 @@
-package oop.proxy
+package oop.Proxy
```

Comment:
Yes I have see the other packages with Mayus, I have never seen a package started with mayus. We need change it.